docs/mpdev-reference.md

```diff
@@ -1,7 +1,3 @@
-## Setup
```

Comment:
Make a reference to the installation instruction in the other README file?

Reply:
Done